src/qthotkey/Makefile

```diff
@@ -0,0 +1,19 @@
+PLUGIN = qthotkey${PLUGIN_SUFFIX}
+
+SRCS = plugin.cc gui.cc moc_plugin.cc moc_gui.cc
+
+include ../../buildsys.mk
+include ../../extra.mk
+
+plugindir := ${plugindir}/${GENERAL_PLUGIN_DIR}
+
+LD = ${CXX}
+CFLAGS += ${PLUGIN_CFLAGS}
+CPPFLAGS += ${PLUGIN_CPPFLAGS} ${QT_CFLAGS} -I../.. -I..
+LIBS += ${QT_LIBS} -lX11
+
+moc_%.cc: %.h
```

Comment:
Using MOC is not allowed in audacious, in most cases you don't actually need it and can achieve the same results with subclassing.

Reply:
To follow up, I don't see any Q_OBJECT macros in your code, so the MOC is entirely unnecessary.

Reply:
There were a few Q_OBJECT macros, but I removed them, and it looks like they were no longer needed since plugin worked fine for me.

Thus I removed moc and qrc support from Makefile.